### ISSUE TITLE
Copy and Paste throws error in KitchenSinkAngular6

### ISSUE THREAD

**Issue by ArthurWietzorek** [clientIO/joint#1196]
After running into some problems while using the `joint.ui.Clipboard`, i wanted to try out my implementation in the KitchenSinkAngular6 demo, to make sure they were not caused by some unrelated problem. After downloading and extracting Rappid 3.0.1, i `npm install`ed and `ng serve`d the app and tried using `ctrl+c` and `ctrl+v` to copy and paste some rectangles. 

Upon pasting, an error is thrown:
![image](https://user-images.githubusercontent.com/41048767/66487972-869a1f00-eaad-11e9-915e-32384555d080.png)

And the invisible rectangles and links are selected:
![image](https://user-images.githubusercontent.com/41048767/66488069-a6c9de00-eaad-11e9-821e-e8c8ca637228.png)

This is very similar to the problem occuring in our project, which got me interested in underlying cause.

**Comment by vtalas** [comment]
@ArthurWietzorek this is, unfortunately, a bug in the `ui.Clipboard` plugin. It will be fixed in the following patch (v3.0.2). Meanwhile, you can workaround this by the `useLocalStorage` set to false in 
 options of the `paste` method: 

```
            const pastedCells = clipboard.pasteCells(graph, {
                translate: { dx: 20, dy: 20 },
                useLocalStorage: false
            });
```

Note that when `localStorage` is switched off, copy&paste won't work among multiple windows. Let us know if this is not acceptable for you.

**Comment by ArthurWietzorek** [comment]
Thank you, this workaround will suffice.

**Comment by kumilingus** [comment]
Fixed in Rappid `v3.1.1`.
